desktop/enterprise/admin/install/mac.md

```diff
@@ -32,7 +34,7 @@ Double-click the `.pkg` file to begin the installation and follow the on-screen
 
 Mac administrators can use the command line option `\$ sudo installer -pkg Docker.pkg -target /` for fine tuning and mass installation. After running this command, you can start Docker Desktop from the Applications folder on each machine.
 
-Administrators can configure additional settings by modifying the administrator configuration file. For more information, see [Configure Desktop Enterprise for Mac](/ee/desktop/admin/configure/mac-admin).
+Administrators can configure additional settings by modifying the administrator configuration file. For more information, see [Configure Desktop Enterprise for Mac](/desktop/enterprise/admin/configure/mac-admin).
```

Comment:
nit: these URLs should have a trailing `/` - saw a couple of other ones as well (we can update in a follow-up and link to the `.md` file)

Reply:
Sure, I only changed the initial path. Happy to follow up in a separate commit.